modules/Audit/Audit.php

```diff
@@ -126,7 +126,8 @@ public function get_audit_list()
 
         if ($focus->is_AuditEnabled()) {
             $order= ' order by '.$focus->get_audit_table_name().'.date_created desc' ;//order by contacts_audit.date_created desc
-            $query = "SELECT ".$focus->get_audit_table_name().".*, users.user_name FROM ".$focus->get_audit_table_name().", users WHERE ".$focus->get_audit_table_name().".created_by = users.id AND ".$focus->get_audit_table_name().".parent_id = '$focus->id'".$order;
+            $unknown_label = translate('LBL_UNKNOWN', 'Users');
```

Comment:
You actually wouldn't need to use the translate function as this will get done through the language packs hosted on crowdin.

```suggestion
            $unknown_label = $mod_strings['LBL_UNKNOWN'];
```

Make sure to add global $mod_strings as well.

Reply:
@Dillon-Brown Thanks for CR.
1. Your suggested doesn't work for me in 7.9.7 I think its because `LBL_UNKNOWN` is missing in `Audit` language file. (I added $mod_strings to the global declaration)

1. I can add `LBL_UNKNOWN` to Audit language file but don't you think that fore use translation from the `User` module its the right translation for this purpose?